README.md

```diff
@@ -92,7 +104,17 @@ The Admin sign in through sign-in page and he will be redirected to the dashboar
 
 ## The MVP :rainbow:
 
- GIF Image
+ - Web
+ 
+![](https://i.imgur.com/KtYlkTi.gif)
+
+![](https://i.imgur.com/CaNT7yA.gif)
```

Comment:
In web it should be two GIFS one for the dashboard and the other for the applicant, I just see one here